### ISSUE TITLE
GCP aliasIP only moves on NIC0, need aliasIP to move on other NICs too

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-cloud-failover-extension#32]
## Request
Currently, CFE in GCP only moves aliasIP ranges associated with NIC0. I have a use case for aliasIPs on NIC2 (aka "internal network") to move as well. This request is to have CFE look for all data plane NICs and move aliasIPs if found.

## Current Behavior
1. deploy HA via API 3nic template in GCP with aliasIP (this gets assigned to NIC0)
2. once deployed, manually go to active BIG-IP VM and add new aliasIP range to NIC2
3. test failover
4. Only NIC0 aliasIP ranges move to new active unit
5. NIC2 aliasIP ranges remain on old active unit

## Expected Behavior
1. failover should move all aliasIP ranges associated with all data plane NICs

## Use Case
Outbound SNAT needs to have floater SNAT. BIG-IP cluster sending egress traffic needs to have consistent floater IP during failover events. Without an aliasIP assigned to the NIC, traffic will not leave on the additional IP assigned to the SNAT if the VM only has one self IP assigned.

**Comment by JeffGiroux** [comment]
During CFE query, it does see the VM and it does see the extra aliasIP.

snippet...
```
Thu, 25 Jun 2020 21:39:50 GMT - fine: [f5-cloud-failover] Cloud provider found vms: {"0":{
...
{"network":"https://www.googleapis.com/compute/v1/projects/xxxxx/global/networks/jgiroux-net-int","subnetwork":"https://www.googleapis.com/compute/v1/projects/xxxxx/regions/us-west1/subnetworks/jgiroux-subnet-int","networkIP":"10.1.20.39","name":"nic2","aliasIpRanges":[{"ipCidrRange":"10.1.20.222/32"}],
```

Then the end of logs only shows NIC0 aliasIPs moved...
```
Thu, 25 Jun 2020 21:39:53 GMT - finest: [f5-cloud-failover] _updateAddresses interface operations:  {"disassociate":[["bigip1-jg-f5-api-ha","nic0",{"aliasIpRanges":[],"fingerprint":"Vf_AbYiKV24="},{"zone":"us-west1-a"}]],"associate":[["bigip2-jg-f5-api-ha","nic0",{"aliasIpRanges":[{"ipCidrRange":"10.1.10.196/32"}],"fingerprint":"zyb8c4y8zYY="},{"zone":"us-west1-b"}]]}
Thu, 25 Jun 2020 21:39:53 GMT - finest: [f5-cloud-failover] _updateAddresses forwarding rules operations:  {"operations":[["jg-f5-api-ha-fr0","https://www.googleapis.com/compute/v1/projects/xxxxx/zones/us-west1-b/targetInstances/bigip2-jg-f5-api-ha-ti"]]}
Thu, 25 Jun 2020 21:39:53 GMT - finest: [f5-cloud-failover] updateAddresses disassociate operations:  {"0":["bigip1-jg-f5-api-ha","nic0",{"aliasIpRanges":[],"fingerprint":"Vf_AbYiKV24="},{"zone":"us-west1-a"}]}
Thu, 25 Jun 2020 21:39:53 GMT - finest: [f5-cloud-failover] updateAddresses associate operations:  {"0":["bigip2-jg-f5-api-ha","nic0",{"aliasIpRanges":[{"ipCidrRange":"10.1.10.196/32"}],"fingerprint":"zyb8c4y8zYY="},{"zone":"us-west1-b"}]}
Thu, 25 Jun 2020 21:39:53 GMT - finest: [f5-cloud-failover] Updating NIC: nic0 for VM bigip1-jg-f5-api-ha in zone us-west1-a
Thu, 25 Jun 2020 21:39:53 GMT - finest: [f5-cloud-failover] Updating forwarding rule: jg-f5-api-ha-fr0 to target: https://www.googleapis.com/compute/v1/projects/xxxxx/zones/us-west1-b/targetInstances/bigip2-jg-f5-api-ha-ti
Thu, 25 Jun 2020 21:39:54 GMT - finest: [f5-cloud-failover] updateFwdRule operation name: operation-1593121193743-5a8ef6ed65664-e8509f16-06b6b5a2
Thu, 25 Jun 2020 21:39:57 GMT - info: [f5-cloud-failover] Disassociate NICs successful.
Thu, 25 Jun 2020 21:39:57 GMT - finest: [f5-cloud-failover] Updating NIC: nic0 for VM bigip2-jg-f5-api-ha in zone us-west1-b
Thu, 25 Jun 2020 21:40:02 GMT - info: [f5-cloud-failover] Associate NICs successful.
Thu, 25 Jun 2020 21:40:19 GMT - info: [f5-cloud-failover] Updated forwarding rules successfully
Thu, 25 Jun 2020 21:40:19 GMT - finest: [f5-cloud-failover] Data will be uploaded to f5cloudfailoverstate.json:  {"taskState":"SUCCEEDED","message":"Failover Complete","timestamp":"2020-06-25T21:40:19.557Z","instance":"bigip2-jg-f5-api-ha.c.xxxxx.internal","failoverOperations":{"addresses":{"publicAddresses":{},"interfaces":{"disassociate":[["bigip1-jg-f5-api-ha","nic0",{"aliasIpRanges":[],"fingerprint":"Vf_AbYiKV24="},{"zone":"us-west1-a"}]],"associate":[["bigip2-jg-f5-api-ha","nic0",{"aliasIpRanges":[{"ipCidrRange":"10.1.10.196/32"}],"fingerprint":"zyb8c4y8zYY="},{"zone":"us-west1-b"}]]},"loadBalancerAddresses":{"operations":[["jg-f5-api-ha-fr0","https://www.googleapis.com/compute/v1/projects/xxxxx/zones/us-west1-b/targetInstances/bigip2-jg-f5-api-ha-ti"]]}},"routes":{"operations":[]}}}
Thu, 25 Jun 2020 21:40:19 GMT - info: [f5-cloud-failover] Failover Complete
Thu, 25 Jun 2020 21:40:20 GMT - finest: [f5-cloud-failover] Download stateFile: {"taskState":"SUCCEEDED","message":"Failover Complete","timestamp":"2020-06-25T21:40:19.557Z","instance":"bigip2-jg-f5-api-ha.c.xxxxx.internal","failoverOperations":{"addresses":{"publicAddresses":{},"interfaces":{"disassociate":[["bigip1-jg-f5-api-ha","nic0",{"aliasIpRanges":[],"fingerprint":"Vf_AbYiKV24="},{"zone":"us-west1-a"}]],"associate":[["bigip2-jg-f5-api-ha","nic0",{"aliasIpRanges":[{"ipCidrRange":"10.1.10.196/32"}],"fingerprint":"zyb8c4y8zYY="},{"zone":"us-west1-b"}]]},"loadBalancerAddresses":{"operations":[["jg-f5-api-ha-fr0","https://www.googleapis.com/compute/v1/projects/xxxxx/zones/us-west1-b/targetInstances/bigip2-jg-f5-api-ha-ti"]]}},"routes":{"operations":[]}}}
```

**Comment by f5-chidambaram** [comment]
Hi Jeff, thanks for reporting this. I have added this to the backlog, internal issue ID AUTOSDK-377.

**Comment by shyawnkarim** [comment]
Closing. 

This enhancement was included with [Release 1.6](https://github.com/F5Networks/f5-cloud-failover-extension/releases/tag/v1.6.0).
